releasenotes/notes/cws-remove-watch-dir-option-f8faee4937353316.yaml

```
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+deprecations:
+  - |
+    CWS: the `runtime_security_config.policies.watch_dir` option has been removed.
+    Please use remote configuration for dynamically updating policies, or simply send
+    the SIGHUP signal to the system-probe process to reload the policies.
```

```suggestion
    the `SIGHUP` signal to the `system-probe` process to reload the policies.
```

Assuming these are both code literals